ui/AddManager.php

```diff
@@ -317,17 +309,11 @@ public function addManagerMain() {
             $this->addManagerDel();
             break;
         default:
-            $displayDate = date("l, j F Y");
-            echo "  <TABLE WIDTH=\"100%\">\n    <TR><TH ALIGN=LEFT>";
-            echo "A-File as of $displayDate</TH>\n";
-            if($this->session->isAuth("n") || $this->session->isAuth("o")) {
-                echo "        <TD ALIGN=RIGHT VALIGN=TOP><A CLASS=\"sub\" HREF=\"#top\" onClick='window.open(\"?target=afile\")'>Printable A-File</A></TD>\n";
-            }
```

Comment:
You took out the 'Printable A-File' link.  That needs to go back in for the users with the appropriate rights (n or o).

Reply:
done.